webxdc.d.ts

```diff
@@ -97,6 +97,15 @@ interface Webxdc<T> {
    * @returns returns a promise that never resolves (because the xdc closes), but is rejected on error.
    */
   sendToChat(content: sendOptions): Promise<void>;
+  /**
+   * TODO
+   */
+  importFiles(filters: {
```

Comment:
I am not sure about this "filters" object, ex. "multiple" is about selection not really a filter, a better name doesn't comes to my mind tho